server/neptune/workflows/internal/pr/receiver/revision.go

```diff
@@ -0,0 +1,63 @@
+package receiver
+
+import (
+	"github.com/runatlantis/atlantis/server/neptune/workflows/activities/github"
+	"github.com/runatlantis/atlantis/server/neptune/workflows/activities/terraform"
+	"github.com/runatlantis/atlantis/server/neptune/workflows/internal/deploy/request"
+	"github.com/runatlantis/atlantis/server/neptune/workflows/internal/deploy/request/converter"
+	workflowMetrics "github.com/runatlantis/atlantis/server/neptune/workflows/internal/metrics"
+	"go.temporal.io/sdk/temporal"
+	"go.temporal.io/sdk/workflow"
+)
+
+const (
+	TerraformRevisionSignalID = "new-terraform-revision"
+)
+
+type RevisionReceiver struct {
```

Comment:
Doesn't make sense to me to have this in a separate package.  receiver doesn't really tell me anything tbh.

I would move this file to revision and just call this `Receiver`